src/components/ProfileDropdown.jsx

```diff
@@ -18,33 +18,48 @@ import DoneIcon from '@material-ui/icons/Done';
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 
 // hacky way to override Menu style
-const StyledMenu = withStyles({
+const StyledMenu = withStyles((theme) => ({
   paper: {
     border: '1px solid black',
-    width: '300px',
-    background: '#f2f5fa',
+    width: '350px',
+    //background: theme.palette.background.default,
```

Comment:
We had talked about this before and we had to do `background: '#F3F6FA'` if you recall this was done because the `theme.palette.background.default` has the opacity included on the hex code you used

Reply:
Right, you can update the default background color in `theme.js` to the color you're hardcoding so that `theme.palette.background.default` doesn't have an opacity issue.